src/status_im2/contexts/activity_center/notification/reply/view.cljs

```diff
@@ -21,11 +22,19 @@
 (defn get-message-content
   [{:keys [content-type] :as message}]
   (case content-type
-    constants/content-type-text    (get-in message [:content :text])
+    constants/content-type-text        (get-in message [:content :text])
 
-    constants/content-type-image   [old-message/message-content-image message]
+    constants/content-type-image       [old-message/message-content-image message]
 
-    constants/content-type-sticker [old-message/sticker message]))
+    constants/content-type-sticker     [old-message/sticker message]
+
+    ;; NOTE: The following type (system-text) doesn't have a design yet.
+    ;; We added the support for this type temporarily to prevent crashing of AC.
```

Comment:
It would be nice to add the related issue link here. Another option would be to remove the comment and just rely on the GH issue to keep track of the work left to be done.